packages/dep.go

```diff
@@ -14,38 +14,25 @@
 package packages
 
 import (
-	"fmt"
-	"github.com/sonatype-nexus-community/nancy/customerrors"
-	"github.com/sonatype-nexus-community/nancy/types"
-	"os"
+	"github.com/golang/dep"
 	"strings"
 )
 
-// Dep is an implementation of Packages interface
-type Dep struct {
-	ProjectList types.ProjectList
-	GopkgPath   string
-}
-
 // ExtractPurlsFromManifest will convert Gopkg projects to Package URLs
-func (d Dep) ExtractPurlsFromManifest() []string {
+func ExtractPurlsUsingDep(project dep.Project) []string {
```

Comment:
Comment is out of date, and the `UsingDep` suffix could probably be dropped (I think the type covers that sufficiently now).